bin/lamassu-update

```diff
@@ -4,6 +4,9 @@ set -e
 export LOG_FILE=/tmp/update.$(date +"%Y%m%d").log
 export NPM_BIN=$(npm -g bin)
 
+MNEMONIC_FILE=$HOME/mnemonics/mnemonic.txt
```

Comment:
* Should be based off of lamassu.json
* Should update lamassu.json
* Path should be ``/etc/lamassu/mnemonics/mnemonic.txt``

Reply:
The `lamassu.json` update happens on `lamassu-update-to-mnemonic` script.

> Should be based off of lamassu.json
> Path should be /etc/lamassu/mnemonics/mnemonic.txt

I don't get it, should I create a script to get the path from `lamassu.json` and default the value to `/etc/lamassu/mnemonics/mnemonic.txt` ?

Reply:
In this case, you should run lamassu-update-to-mnemonic every time. lamassu-update-to-mnemonic should gracefully fail when the mnemonic file is present. That way, this script (lamassu-update) doesn't have to deal with mnemonic path logic.

Another issue is that lamassu-update-to-mnemonic is used for both developers and production. In production, we want to write to ``/etc/lamassu/mnemonics`` and for developers we probably want the path to be ``~/.lamassu/mnemonics``. So the script should be able to handle both cases depending on who is calling it.